src/main/java/org/broadinstitute/hellbender/tools/walkers/ReferenceConfidenceVariantContextMerger.java

```diff
@@ -615,25 +764,102 @@ private int findHomIndex(final int i, final int ploidy, final int numAlleles) {
      * @param indexesOfRelevantAlleles the indexes of the original alleles corresponding to the new alleles
      * @return non-null array of new AD values
      */
+    public static int[] generateAD(final int[] originalAD, final int[] indexesOfRelevantAlleles) {
+        final List<Integer> adList = (List<Integer>)remapRLengthList(Arrays.stream(originalAD).boxed().collect(Collectors.toList()), indexesOfRelevantAlleles);
+        final int[] ads = new int[adList.size()];
```

Comment:
`return Ints.toArray(adList)`

Reply:
I keep forgetting that we use Guava.  Good stuff.